src/NUnitEngine/nunit.engine.api/Extensibility/IFrameworkDriver.cs

```diff
@@ -39,7 +40,7 @@ public interface IFrameworkDriver
         /// used to ensure that test ids are unique across drivers.
         /// </summary>
         string ID { get; set; }
-
+        
```

Comment:
Today I also discovered a VS Code extension for this too (https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces) which is totally awesome. I enabled [trim on save](https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces#trim-on-save).